### PR TITLE
Add Brand for each community

### PR DESCRIPTION
**Pull request by filosganga**

### CODE REVIEW

File: init-twilio-sids.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+credstash put uat.twilio.service_sid.energy_sw MG665c75567687673591e8f174286a3034
```
Comment (laurence-bird) [inline init-twilio-sids.sh]: did you mean to commit this?

Comment (filosganga) [reply init-twilio-sids.sh]: Nope, it is a leftover

Comment (filosganga) [reply init-twilio-sids.sh]: Fixed

**Comment by ovo-comms-circleci** [comment]
This PR was successfully deployed to production by [CircleCI build 542](https://circleci.com/gh/ovotech/comms-delivery-service/542) at 2018-10-18 17:40 BST.

**Pull request merged**